plankton/adapters/__init__.py

```diff
@@ -133,6 +138,12 @@ def __init__(self, target_member, property_name):
         self._target_member = target_member
         self._prop = property_name
 
+        try:
+            self.__doc__ = getattr(type(getattr(instance, self._target_member)),
+                                   self._prop).__doc__
```

Comment:
`instance` is required for actually retrieving the docstring, because the property can not be bound to a specific instance at this point. In principle it would be possible to take `type(instance)` instead, but it seemed easier to just ask for an object (which is `self` in the only place `ForwardProperty` is used).